Tests/ShamTests/Tests/StubRequestTests.swift

```diff
@@ -6,10 +6,24 @@ import UtilityBeltNetworking
 import XCTest
 
 final class StubRequestTests: XCTestCase {
-    func testRandomizedQueryStringOrderComparisonSucceeds() {
-        let request: StubRequest = .get("https://spothero.local?shape=triangle&color=blue")
-        let canMockData = request.canMockData(for: .get("https://spothero.local?color=blue&shape=triangle"))
+    
+    private let baseURLString = "https://spothero.local"
+    
+    func testDescriptionWithUnsortedParameters() {
+        let unsortedQueryParametersURL = "\(self.baseURLString)?zebra=thing&ardvark=other_thing"
+        let request: StubRequest = .get(unsortedQueryParametersURL)
         
-        XCTAssertTrue(canMockData, "Unable to mock data for request.")
+        let sortedQueryParametersURL = "\(self.baseURLString)?ardvark=other_thing&zebra=thing"
+        XCTAssertEqual(request.description, "get: \(sortedQueryParametersURL)")
+    }
+    
+    func testDescriptionWithNoParameters() {
+        let request: StubRequest = .post(self.baseURLString)
+        XCTAssertEqual(request.description, "post: \(self.baseURLString)")
```

Comment:
Surprised this passes, the method should print out as `POST`. 🤔 

```suggestion
        XCTAssertEqual(request.description, "POST: \(self.baseURLString)")
```

Reply:
thats how I had it originally, and tests failed because the methods were lowercased... I wouldve guessed the same as you, given the raw values are `case post = "POST"`.... not sure why they are lowercased?

Reply:
Added a suggestion to fix this above since it's not printing the rawValue, so this can be committed when that is as well.